modules/user_accounts/templates/menu_user_accounts.tpl

```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```

Comment:
Yeah I'm happy to have input on the design decisions. I just think it looks bad/weird the way it is but I'm not married to any one solution.